sourcetracker/_cli/gibbs.py

```diff
@@ -105,6 +102,15 @@
                     'has been successfully started, the jobs will not be '
                     'successfully launched. If this is happening, increase '
                     'this parameter.'))
+@click.option('--full_output', required=False, default=False,
+              is_flag=True, show_default=True,
+              help=('Write the `full output` of a run. If True, this option '
+                    'will cause SourceTracker2 to write out a feature table '
+                    'for each sink (or source if `--loo` is passed. These '
+                    'feature tables contain the specific sequences that '
+                    'contributed to a sink from a given environment. This '
+                    'option can be memory intensive if there are a large '
```

Comment:
I think so, but am not sure. If we have 100 sinks, 200 sources, and 20,000 features, we have 8e8 entries to process and write.